source/includes/run-command.kt

```diff
@@ -0,0 +1,23 @@
+import com.mongodb.kotlin.client.MongoClient
+import org.bson.Document
+
+fun main() {
+    //start-execute
+    val explanation = database.runCommand({
+            explain: {
+                find: 'restaurants'
+            }
+        })
+    //end-execute
+
+    //start-read-preference
+    val command = { "hello": 1 }
+    val commandReadPreference = { readPreference: "secondary" }
```

Comment:
I think there's a method for this

```suggestion
    val commandReadPreference = ReadPreference.secondary()
```